NEWS.md

```diff
@@ -61,6 +61,7 @@
 
 ### RStudio Server Pro
 
+* **DEPRECATED:** The option `auth-proxy-require-hmac` is no longer operational and a warning will be issued to the logs if used. See the documentation on proxy authentication for alternatives. (Pro #2029)
```

Comment:
Nit: let's not lead with this as the very first thing people read about what we've done in this release! (put it towards the end of the notes)